src/Components/containers/ValueBlockBirds/ValueBlockBirds.tsx

```diff
@@ -0,0 +1,125 @@
+import './ValueBlockBirds.style.css'
+import { DescriptionBirdDefault } from '../../view/DescriptionBirdDefault/DescriptionBirdDefault'
+import { useState } from 'react'
+import { BtnNextLevel } from '../../view/BtnNextLevel/BtnNextLevel'
+import { DescriptionBirdValue } from '../../view/DescriptionBirdValue/DescriptionBirdValue'
+import useSound from 'use-sound'
+
+import { alarmError } from '../../constans/alarmValue'
+import { alarmRight } from '../../constans/alarmValue'
+import { ADD_POINTS } from '../../constans/reducerActionType'
+import { useDispatch } from 'react-redux'
+import { IBird } from '../../constans/birds.interface'
+import { IPropsValueBirds } from './ValueBlockBirds.interface'
+
+const ValueBlockBirds: React.FC<IPropsValueBirds> = ({
+  birds,
+  url,
+  setMainBird,
+  setPercentage,
+  percentage,
+}) => {
+  const [item, addItems] = useState<IBird[]>([])
+  const [disabled, changeDisabled] = useState(true)
+  const [styleBtn, changeStyleBtn] = useState('nexBtn')
+  const [errorState, changeErrorState] = useState('error')
+  const [points, setPoints] = useState(5)
+  const [audioControl, setAudioControl] = useState(0)
+  const [playError] = useSound(alarmError)
+  const [playRight] = useSound(alarmRight)
+
+  const dispatch = useDispatch()
+
+  const addPoints = (points: number) => {
+    dispatch({ type: ADD_POINTS, payload: points })
+  }
+  const liOnClick = (e: React.MouseEvent<HTMLLIElement, MouseEvent>) => {
+    setAudioControl(0)
+    const target = e.target as HTMLInputElement
+    const findElement: IBird | undefined = birds?.find(
+      (item: IBird) => item.id === Number(target.id)
+    )
+    if (findElement) {
+      addItems([findElement])
+    }
+    if (!findElement?.value) {
+      points ? setPoints(points - 1) : setPoints(0)
+    }
+
+    if (findElement?.value) {
+      const mainBird: IBird = birds.filter((bird: IBird) => bird.value)[0]
```

Comment:
Можно достать первый элемент через деструктуризацию

```suggestion
      const [mainBird] = birds.filter((bird: IBird) => bird.value)
```